src/ppl/src/Netlist.cpp

```diff
@@ -184,6 +185,7 @@ void Netlist::clear()
   io_pins_.clear();
   io_groups_.clear();
   _db_pin_idx_map.clear();
+  net_pointer_.push_back(0);
```

Comment:
Why is this required?

Reply:
This is required because I've moved the call for the Netlist::clear() to the initialization of the netlist data. The `net_pointer_` vector is cleared above, but it needs to start with 0.

Perhaps renaming this function to Netlist::reset() will make things clearer since the idea is to restore the Netlist state to the beginning of the PPL flow execution in cases where PPL is called multiple times in a single run.